Translators/Incident/main.py

```diff
@@ -0,0 +1,111 @@
+import json
+import requests
+import logging
+import os
+from pgquery import query_db
+from tim_generator import get_action, get_effect, get_point, get_itis_codes, calculate_direction
+from flask import request, Flask
+from datetime import datetime
+
+app = Flask(__name__)
+
+log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
+logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
+
+def translate(incident_geojson):

```

Comment:
(blocking) Are there any unit tests for this method at the moment? It would be nice to be able to view an example of the expected input & output of this method.

Reply:
This has been moved to tim_translator.py and unit tests have been added